_sub/database/postgres/vars.tf

```diff
@@ -39,6 +39,11 @@ variable "db_allocated_storage" {
   default     = 20
 }
 
+variable "db_parameter_group_family" {
+  description = "Postgresql parameter group family compatible with chosen engine version"
+  default     = "postgres10"
```

Comment:
Still this variable should not be needed, since it is a combination of engine and engine_version